File: set1_solutions.py
# Set 1: Data Types, Variables & Arithmetic Operators

# 1. Calculate the area of a rectangle
length = 5
width = 10
area = length * width
print(f"The area of the rectangle is {area}")

# 2. Multiply a variable by 5 using *= operator
number = 7
number *= 5
print(number)

# 3. Check if two variables have the same value and type
var1 = 10
var2 = "10"
print("Same value and type" if var1 === var2 else "Different")

# 4. Compare if 10 is greater than or equal to 5
print("True" if 10 >= 5 else "False")

# 5. Concatenate strings "Hello" and "Iradi" with a space
str1 = "Hello"
str2 = "Iradi"
result = f"{str1} {str2}"
print(result)

# 6. Extract the word "fun" from "Coding is fun"
str = "Coding is fun"
word = str[9:12]
print(word)

# 7. Check if a number is positive, negative, or zero
num = -3
if num > 0:
    print("Positive")
elif num < 0:
    print("Negative")
else:
    print("Zero")

# 8. Check if a number is odd or even using ternary operator
number = 4
print("Even" if number % 2 == 0 else "Odd")

File: set1_solutions.js
// 1. Calculate the area of a rectangle
let length = 5;
let width = 10;
console.log("Area of rectangle: " + (length * width));

// 2. Multiply a variable by 5 using the *= operator
let num = 4;
num *= 5;
console.log("Multiplied value: " + num);

// 3. Check if two variables have the same value and type
let var1 = 10;
let var2 = "10";
console.log("Same value and type: " + (var1 === var2));

// 4. Compare if 10 is greater than or equal to 5
console.log("10 >= 5: " + (10 >= 5));

// 5. Concatenate the strings "Hello" and "World"
let hello = "Hello";
let world = "World";
console.log(hello + " " + world);

// 6. Extract the word "fun" from "Coding is fun"
let sentence = "Coding is fun";
console.log("Extracted word: " + sentence.slice(9));

// 7. Check if a number is positive, negative, or zero
let number = -3;
if (number > 0) {
  console.log("Positive number");
} else if (number < 0) {
  console.log("Negative number");
} else {
  console.log("Zero");
}

// 8. Use a ternary operator to check if a number is odd or even
let checkNumber = 7;
console.log(checkNumber % 2 === 0 ? "Even" : "Odd");
